resources/models/reservation.py

```diff
@@ -419,16 +421,28 @@ def send_reservation_denied_mail(self):
         self.send_reservation_mail(NotificationType.RESERVATION_DENIED)
 
     def send_reservation_confirmed_mail(self):
-        self.send_reservation_mail(NotificationType.RESERVATION_CONFIRMED)
+        reservations = [self]
+        ical_file = build_reservations_ical_file(reservations)
+        attachment = 'reservation.ics', ical_file, 'text/calendar'
```

Comment:
This is a tuple, I would use parenthesis to make it clear
`('reservation.ics', ical_file, 'text/calendar')`

Reply:
Done.